### ISSUE TITLE
CheckBox control, righttoleft property

### ISSUE THREAD

**Issue by dddns** [sobatdata/Guna.UI-Framework-Lib#34]
Howdy,

If I use righttoleft property... the design... goes away with the default .net fram
![checkbox, righttoleft](https://user-images.githubusercontent.com/13902147/81743648-ef110a80-9499-11ea-837f-a23e944a1cff.jpg)
ework, check the image.

**Comment by sobatdata** [comment]
We're sorry, we will fix it soon.

Thank you

**Comment by sobatdata** [comment]
Another solution you can use Guna2CustomCheckBox.

Thank you

**Comment by dddns** [comment]
Hi,

Is this fix, related to this reported issue?

Fixed - Bug when the RihtToLeft property was given the value 'Yes' in controls: Guna2RadioButton, Guna2CheckBox and Guna2TextBox

If yes, is it possible to close the bugs when you solve them?
Thanks,

**Comment by sobatdata** [comment]
Yes it is the bug you reported and I have fixed the bug. thank you very
much for always reporting new bugs to me.

Pada tanggal Sen, 25 Mei 2020 22.27, dddns <notifications@github.com>
menulis:
